__tests__/app.test.js

```diff
@@ -83,4 +81,57 @@ describe("/api/articles/:article_id",()=>{
       expect(msg).toBe("Bad Request")
     })
   })
+})
+
+
+
+describe("GET /api/articles",()=>{
+  test("200 :Responds with an articles array of article objects, each of which should have the following properties: author,title,article_id,topic,created_at,votes,article_img_url and comment_count (the total count of all the comments with this article_id)",()=>{
```

Comment:
For test descriptions you dont need to write out all of the expected properties, since this is already written in your expect statements below, you have already described the endpoint and that part can be removed, but with the exception of comment_count as you are explaining what this value means which I think is nice